install/kubernetes/cilium/templates/hubble-cli.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.hubble.cli.enabled }}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: hubble-cli
+  labels:
+    k8s-app: hubble-cli
+  namespace: {{ .Release.Namespace }}
+spec:
+  containers:
+  - name: hubble-cli
+    image: quay.io/cilium/hubble:latest
+    command:
+      - sleep
+      - "3600"
```

Comment:
i think some people (myself included) will want to keep the cli pod around. maybe do something like:
```
tail -f /dev/null
```
instead of exiting after 1 hour.